commerce-price-list-service/service.xml

```diff
@@ -99,8 +99,8 @@
 		<finder name="CompanyId" return-type="Collection">
 			<finder-column name="companyId" />
 		</finder>
-		<finder name="CommerceCurrencyId" return-type="Collection">
-			<finder-column name="commerceCurrencyId" />
+		<finder name="CommerceCurrencyCode" return-type="Collection">
```

Comment:
Re-sorting the `commerceCurrencyCode` column will mean this finder will need to be re-sorted as well.